packages/contracts/src/dollar/core/DollarMintCalculator.sol

```diff
@@ -20,17 +20,16 @@ contract DollarMintCalculator is IDollarMintCalculator {
         manager = UbiquityDollarManager(_manager);
     }
 
-    /// @notice returns (TWAP_PRICE  -1) * Ubiquity_Dollar_Total_Supply
+    /// @notice returns (TWAP_PRICE  -1) * UAD_Total_Supply
```

Comment:
```suggestion
    /// @notice returns (TWAP_PRICE  -1) * Dollar_Total_Supply
```

We should never refer to the tokens by their symbols in the codebase because they are not as clear for new developers as writing their utility (Dollar, Credit etc) also we can (and already have) updated the symbols.

Reply:
done